apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java

```diff
@@ -522,31 +530,31 @@ public ShufflingParticipant getLastParticipant() {
                 if (!Crypto.isCanonicalPublicKey(publicKey) || !recipientAccounts.add(Account.getId(publicKey))) {
                     // duplicate or invalid recipient public key
                     LOG.debug("Invalid recipient public key " + Convert.toHexString(publicKey));
-                    return new Attachment.ShufflingRecipients(this.id, Convert.EMPTY_BYTES, shufflingStateHash);
+                    return new ShufflingRecipients(this.id, Convert.EMPTY_BYTES, shufflingStateHash);
                 }
             }
             // last participant prepares ShufflingRecipients transaction instead of ShufflingProcessing
-            return new Attachment.ShufflingRecipients(this.id, outputDataList.toArray(new byte[outputDataList.size()][]),
+            return new ShufflingRecipients(this.id, outputDataList.toArray(new byte[outputDataList.size()][]),
```

Comment:
Maybe add suffix `Attachment` to all classes, which implement Attachment interface?